KotlinMMP/iosApp/iosApp/ViewModels/Color.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+
+extension Color {
+    init?(hex: String) {
+        var hexSanitized = hex.trimmingCharacters(in: .whitespacesAndNewlines)
+        hexSanitized = hexSanitized.replacingOccurrences(of: "#", with: "")
+
+        var rgb: UInt64 = 0
+
+        var red: Double = 0.0
+        var green: Double = 0.0
+        var blue: Double = 0.0
+        var opacity: Double = 1.0
+
+        let length = hexSanitized.count
+
+        guard Scanner(string: hexSanitized).scanHexInt64(&rgb) else { return nil }
+
```

Comment:
Some magic numbers in the section below, maybe give them variables so this is easier to read?